benchmarks/flash_and_check.py

```diff
@@ -52,6 +59,9 @@ def await_output(s: serial.Serial, target: str):
         await_output(serial, "START\n")
         startTime = time.monotonic()
         print("START found, waiting for DONE", file=sys.stderr)
-        await_output(serial, "DONE\n")
-        endTime = time.monotonic()
-        print(endTime - startTime)
+        success = await_output(serial, "DONE\n", failure="Guru Meditation Error")
```

Comment:
Guru Meditation Error?

Reply:
when wasm3 runs out of stack space

Reply:
Is that a string Wasm3 returns in this case?

Reply:
No that's the standard error on the ESP32